src/openforms/authentication/fields.py

```diff
@@ -17,6 +17,35 @@ def __init__(self, *args, **kwargs):
         self.validators.append(UniqueValuesValidator())
 
 
+class AuthenticationBackendChoiceField(CharField):
```

Comment:
Do we still need those form fields after removing the non-react admin change form? I think this entire class can be safely removed.

Reply:
I think we can, though I'll have to check how to fix the existing migrations for DH

Reply:
yeah there are no constraints added or anything, so you should be able to un-apply the migration. Faking should not be needed. You can check the docs in our internal wiki page, I linked it on Slack this week.